packages/sol-compiler/src/compiler.ts

```diff
@@ -65,6 +82,32 @@ export class Compiler {
     private readonly _artifactsDir: string;
     private readonly _solcVersionIfExists: string | undefined;
     private readonly _specifiedContracts: string[] | TYPE_ALL_FILES_IDENTIFIER;
+    private static async _getSolcAsync(
+        solcVersion: string,
+    ): Promise<{ solcInstance: solc.SolcInstance; fullSolcVersion: string }> {
+        if (_.isUndefined(binPaths[solcVersion])) {
+            throw new Error(`${solcVersion} is not a known compiler version`);
+        }
+        const fullSolcVersion = binPaths[solcVersion];
```

Comment:
Rather then use `binPaths[solcVersion]` twice, lets declare `fullSolcVersion` higher up and reference it in two places.